src/igvfd/loadxl.py

```diff
@@ -51,10 +51,10 @@
     'auxiliary_set',
     'prediction',
     'reference_file',
+    'configuration_file',
```

Comment:
if the goal is to load fist seqspec before sequencing file then the proper way to do it is pipeline 1 pipeline 2 below.

Reply:
fixed